WBooks/AddNew/AddNewViewController.swift

```diff
@@ -10,7 +10,17 @@ import UIKit
 class AddNewViewController: UIViewController {
     
     private lazy var addNewView = AddNewView()
-
+    private let _addNewViewModel: AddNewViewModel
```

Comment:
Why do we add the "_" at the beginning of the propertie?

Reply:
take it as a reference in the example that was mentioned in the training of the card

Reply:
@danielcharry @gmazzei-wolox, What do you think?